src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -1,99 +1,71 @@
 // This component will be modified according to the new wirefrime
 
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
 import {
-  FormGroup, ModalDialog, useToggle, Form,
+  FormGroup, ModalDialog, Form, ModalCloseButton,
 } from '@edx/paragon';
 import { Button } from 'react-paragon-topaz';
-import { handleInstructorsEnrollment } from 'features/Instructors/data/api';
 import { logError } from '@edx/frontend-platform/logging';
-import { fetchClassesData } from 'features/Instructors/data/thunks';
 
-const AddInstructors = () => {
-  const stateInstructors = useSelector((state) => state.instructors.classes);
+import { addInstructor } from 'features/Instructors/data/thunks';
+
+const AddInstructors = ({ isOpen, close }) => {
```

Comment:
I think `close` is not clear enough, maybe `handleOnClose` or `onClose` could be more descriptive.

Reply:
done